ui/cypress/e2e/explorer.test.ts

```diff
@@ -654,8 +654,23 @@ describe('DataExplorer', () => {
           })
         cy.getByTestID('_value-table-header').click()
         cy.get('.table-graph-cell__sort-asc').should('exist')
-        // TODO: complete testing when FE sorting functionality is sorted
-        // https://github.com/influxdata/influxdb/issues/16200
+        cy.getByTestID('_value-table-header').click()
+        cy.get('.table-graph-cell__sort-desc').should('exist')
+        cy.getByTestID('_value-table-header')
+          .should('exist')
```

Comment:
fyi, `cy.get('foo').should('exist')` is almost always superfluous. the `get` operation implicitly asserts existence.

Reply:
Good point, but it is a way of maintaining UI stability in that we want to know whether the UI is manipulated by the click event

Reply:
@Asalem1 

>but it is a way of maintaining UI stability in that we want to know whether the UI is manipulated by the click event

i'm curious how `should('exist')` does that where `cy.getByTestID('_value-table-header')` doesn't?